app/Http/Controllers/CelebController.php

```diff
@@ -41,12 +44,15 @@ public function store(Request $request)
     /**
      * Display the specified resource.
      *
-     * @param  \App\Models\Celeb  $celeb
+     * @param Celeb $celeb
      * @return \Illuminate\Http\Response
      */
     public function show(Celeb $celeb)
     {
-        //
+        $titles = Celeb::with(['movies'])
+                    ->where('id', $celeb->id)
+                    ->get();
```

Comment:
So Kieran, when you are already passing the celeb as an argument you do not need the variable.

Laravel knows that the parameter is a celebrity model, you do not need to call the model again and fetch the record. That will be resolved automatically. 

So for this, you can simplify to:

```
$celeb->with('movies')->get;
```

The `with` will only need array brackets if there is more than one relation. 

Then on the return, you just pass celeb. 

With the code you have implemented here, you are basically passing the same object two times, which means more work for the DB.

So the majority of cases, for the show, update and delete methods you do not fetch the model, just use the one from the request.